test/frontend.js

```diff
@@ -620,6 +620,47 @@ describe('Frontend', function () {
         .exists(elT('inviteMessage'))
       should(success).equal(true)
     })
+    it('Should See Member List on Dashboard', async function () {
+      this.timeout(4000)
+      // login
+      await n
+        .click(elT('openProfileDropDown'))
+        .click(elT('logoutBtn'))
+        // Open login modal
+        .wait(elT('loginBtn'))
+        .click(elT('loginBtn'))
+        // Login
+        .wait(`${elT('loginModal')}.is-active`)
+        .insert(elT('loginName'), username + '5')
+        .insert(elT('loginPassword'), 'testtest')
+        .click(elT('loginSubmit'))
+        .wait(
+          (el) => !document.querySelector(el),
+          `${elT('loginModal')}.is-active`
+        )
```

Comment:
We repeat a lot these steps from l625 to l640 (login). Is it possible in Nightmare to compose "steps", like just calling `.doLogin()` and it reproduces all these steps?

Reply:
👍  I will look into it, it would certainly simplify our tests a lot!